content/docs/guides/crosswalk/aws/iam.md

```diff
@@ -118,7 +118,7 @@ For a full list of available managed policy ARNs, please refer to the
 An AWS Identity and Access Management (IAM) user is an entity that you create in AWS to represent the person or
 application that uses it to interact with AWS. A user in AWS consists of a name and credentials.
 
-Use the [`aws.iam.User` class]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/iam#User" >}}) to create new
+Use the [`aws.iam.User` class]({{< relref "/docs/reference/pkg/aws/iam/user" >}}) to create new
```

Comment:
To @praneetloke's point, perhaps we should generalize the surrounding text a bit. Something like:

```suggestion
Use the [`User` resource]({{< relref "/docs/reference/pkg/aws/iam/user" >}}) to create new
```